README.md

```diff
@@ -11,10 +11,23 @@ There are two options for installing nlp_primitives. Both of the options will al
 
 The first option is to install a version of nlp_primitives that does not include Tensorflow. With this option, primitives that depend on Tensorflow cannot be used. Currently, the only primitive that can not be used with this install option is ``UniversalSentenceEncoder``.
 
+### M1 Macs (via native `arm64` architecture)
```

Comment:
fwiw, the level of this header makes it seem like it's a header for the pypi and conda-forge sections and not an additional step you might need to take 
<img width="458" alt="image" src="https://user-images.githubusercontent.com/64278226/160722321-6d61d488-3bd3-475d-97a1-d9273680be29.png">